stripe/src/test/java/com/stripe/android/PaymentConfigurationTest.java

```diff
@@ -38,19 +38,16 @@ public void getInstance_withPublicKey_returnsDefaultInstance() {
         PaymentConfiguration payConfig = PaymentConfiguration.getInstance();
         assertEquals("pk_test_key", payConfig.getPublishableKey());
         assertEquals(Address.RequiredBillingAddressFields.NONE, payConfig.getRequiredBillingAddressFields());
-        assertTrue(payConfig.getShouldUseSourcesForCards());
     }
 
     @Test
     public void setValues_setsForSingletonInstance() {
         PaymentConfiguration.init("pk_test_key");
         PaymentConfiguration.getInstance()
-                .setRequiredBillingAddressFields(Address.RequiredBillingAddressFields.FULL)
-                .setShouldUseSourcesForCards(false);
+                .setRequiredBillingAddressFields(Address.RequiredBillingAddressFields.FULL);
 
         assertEquals("pk_test_key", PaymentConfiguration.getInstance().getPublishableKey());
         assertEquals(Address.RequiredBillingAddressFields.FULL,
                 PaymentConfiguration.getInstance().getRequiredBillingAddressFields());
-        assertFalse(PaymentConfiguration.getInstance().getShouldUseSourcesForCards());
```

Comment:
I decided to just remove the test for the setting since it's deprecated, but I'm not sure if this is the best approach to take.

Reply:
yeah I think that makes sense